<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtBlockCipherClass</class>
 <widget class="QMainWindow" name="QtBlockCipherClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>498</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>541</width>
    <height>498</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="InputTextField">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>491</width>
      <height>141</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="KeyTextField">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="RunButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="EncryptRadio">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>430</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="DecryptRadio">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>430</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="OutputTextField">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>491</width>
      <height>151</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="OutputLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>38</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output</string>
    </property>
   </widget>
   <widget class="QLabel" name="KeyLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Key</string>
    </property>
   </widget>
   <widget class="QLabel" name="InputLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="InitializationVectorTextField">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="InitializationVectorLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>180</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Initialization vector</string>
    </property>
   </widget>
   <widget class="QComboBox" name="AESModeComboBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>430</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>ECB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CBC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CFB</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="ModeLabel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>430</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="DivideIntoBlocksCheckBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>250</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Divide into blocks</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_key"/>
    <addaction name="actionOpen_iv"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionSave_key_as"/>
    <addaction name="actionSave_iv_as"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSave_key_as">
   <property name="text">
    <string>Save key as</string>
   </property>
  </action>
  <action name="actionOpen_key">
   <property name="text">
    <string>Open key</string>
   </property>
  </action>
  <action name="actionOpen_iv">
   <property name="text">
    <string>Open iv</string>
   </property>
  </action>
  <action name="actionSave_iv_as">
   <property name="text">
    <string>Save iv as</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
